libs/initScripts/local.js

```diff
@@ -18,9 +18,19 @@ const creatorNodeEndpoint1 = 'http://docker.for.mac.localhost:4000'
 const creatorNodeEndpoint2 = 'http://docker.for.mac.localhost:4010'
 const creatorNodeEndpoint3 = 'http://docker.for.mac.localhost:4020'
 const creatorNodeEndpoint4 = 'http://docker.for.mac.localhost:4030'
-const versionStr = '0.3.0'
 const amountOfAuds = 100000
 
+// try to dynamically get versions from .version.json, or use versionStr as a fallback
```

Comment:
@hareeshnagaraj @vicky-g can you take a look at this?

Reply:
in what scenario would we actually hit the fallback? considering .version.json is literally checked in - I think its better to actually just throw out

Reply:
@hareeshnagaraj never should hit this. i could just remove it

Reply:
removed